src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/resource/VisitResourceIntTest.kt

```diff
@@ -67,18 +68,22 @@ class VisitResourceIntTest : IntegrationTestBase() {
 
   @BeforeEach
   internal fun createPrisoner() {
+    threePeople.addAll(
+      (1..3).map {
+        repository.save(PersonBuilder())
+      }
+    )
     offender = repository.save(
       OffenderBuilder()
-        .withBooking(OffenderBookingBuilder().withVisitBalance())
+        .withBooking(
+          OffenderBookingBuilder()
+            .withVisitBalance()
+            .withContacts(*threePeople.map { OffenderContactBuilder(it) }.toTypedArray())
+        )
```

Comment:
this all the above code has achieved - this prisoner now has contacts for each of the visitors :-)